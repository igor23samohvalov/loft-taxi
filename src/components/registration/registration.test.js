import React from 'react';
import { render } from '@testing-library/react';
import Registration from './registration.js';

describe('registration', () => {
    it('renders correctly', () => {
        const { container } = render(<Registration />)
        expect(container).toMatchSnapshot()
    });
});