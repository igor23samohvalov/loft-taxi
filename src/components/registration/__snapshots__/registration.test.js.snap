// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`registration renders correctly 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
    data-testid="authTest"
    style="padding: 40px 55px;"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-content-xs-center MuiGrid-justify-xs-space-between"
      style="height: 436px;"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4"
      >
        Регистрация
      </h4>
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Уже зарегистрированы? 
        <a
          class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          data-id="toAuthorization"
          href="#"
        >
          Войти
        </a>
      </p>
      <form
        action=""
        data-id="toAuthorization"
        href="#"
        id="registration"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-justify-xs-space-between"
          style="height: 220px;"
        >
          <div
            class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
            >
              Адрес электронной почты *
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input"
                placeholder="Адрес электронной почты"
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="MuiGrid-root"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
              style="margin-right: 15px;"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
                data-shrink="false"
              >
                Имя *
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input"
                  placeholder="Имя"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
                data-shrink="false"
              >
                Фамилия *
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input"
                  placeholder="Фамилия"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
            >
              Пароль *
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input"
                placeholder="Пароль"
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </form>
      <div
        class="MuiGrid-root MuiGrid-item"
        style="align-self: flex-end;"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          form="registration"
          tabindex="0"
          type="submit"
        >
          <span
            class="MuiButton-label"
          >
            Зарегистрироваться
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</div>
`;
